pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -89,6 +91,11 @@ def test_serversonly(self):
         suite = BokChoyTestSuite('', serversonly=True)
         self.assertEqual(suite.cmd, "")
 
+    def test_verify_xss(self):
```

Comment:
@efischer19  you should add a test case for the feature being turned on via environment variable.